src/app/locationForm/common/Form.jsx

```diff
@@ -19,7 +19,7 @@ class Form extends Component {
   isEditing(props){
     return props.value === null ||  
       ( typeof props.value === 'object' ? 
-          Object.keys(props.value).some( key => !props.value[key] ) :
+          Object.keys(props.value).some( key => props.value[key] ) :
```

Comment:
Now that I think about it, when I said "other way around", that was pretty confusing. :P
I'm pretty sure this way, we'll actually enter editing mode if we _do_ have at least one value, which is exactly the case where we should be in viewing mode.

What I meant is that I think we should enter editing mode if none of the fields have a value, i.e. changing from `some(!value)`->`every(!value)`, not `some(!value)`->`some(value)`.

Reply:
Fixed in 23a8250468da62b007a56a42c5d65baf6a0cefe7